com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/HDShadowManager.cs

```diff
@@ -48,7 +48,6 @@ struct HDShadowData
         public Vector4      zBufferParam;
         public Vector4      shadowMapSize;
 
-        [SurfaceDataAttributes(precision = FieldPrecision.Real)]
```

Comment:
adding @jessebarker as I guess he may be the one who have done this change.

Reply:
I don't think I added it (hard to tell - git says that line was last edited by you :-) ).  The only issue I can think of is whether both C# and the generated shader struct understand the data as being the same type/layout.  Shouldn't be a problem to take "Real" precision away.